pymeasure/instruments/keithley/keithley6221.py

```diff
@@ -97,6 +97,15 @@ class Keithley6221(Instrument, KeithleyBuffer):
         values=[1e-3, 999999.999],
     )
 
+    output_low_grounded = Instrument.control(
```

Comment:
This file is not part of your this pull request and it is actually part of the master branch.
Again, something wrong with rebase ?